mlmc/sampling_pool.py

```diff
@@ -135,6 +136,24 @@ def change_to_sample_directory(work_dir, path: str):
         sample_dir = os.path.join(work_dir, path)
         if not os.path.isdir(sample_dir):
             os.makedirs(sample_dir, mode=0o775, exist_ok=True)
+
+        # We have observed possible problems with directory creation on the 
```

Comment:
I guessed that there is some problem with NFS, however in fact there was probably problem with various threads writing to the same directory, randomly cleaning it after it was created. 
I suggest to keep the code commented, it allows to better check that we are able to write into the directory. There could be problem with rights anyway.